mkdocs.yml

```diff
@@ -28,6 +28,8 @@ nav:
     - Single Cluster AWS Native Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-native-observability.md
     - Single Cluster Open Source Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability.md
     - Single Cluster Mixed Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md
+    - Single Cluster Graviton Open Source Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-graviton-opensource-observability.md
+    - Single Cluster with no Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-cluster.md
```

Comment:
Just say `Single Cluster`. Also please add the one Existing cluster pattern.

Reply:
done!